docs/react/install.en-US.md

```diff
@@ -31,6 +31,27 @@ $ npm install antd@beta --save
 
 > **Past releases**：https://github.com/ant-design/ant-design/releases
 
+## Import in Browser
+
+We provide `antd.js` `antd.css` and `antd.min.js` `antd.min.css` under `antd/dist` in antd's npm package, in order to import all the components of antd directly. Also, you can use [npmcdn](https://npmcdn.com/).
+
+> It's not a recommended way to use built file, for you cannot get bugfixes from the dependencies of antd.
```

Comment:
> It's not recommended to use the already built files, as you cannot get bugfixes from the dependencies of antd.

TODO: @benjycui I still don't get the whole meaning of this statement. And generally speaking, it would be better to use the positive all the way (instead of "don't to this, as that don't work"), if possible.